crates/biome_grit_formatter/src/grit/declarations/javascript_function_definition.rs

```diff
@@ -0,0 +1,14 @@
+use crate::prelude::*;
+use biome_grit_syntax::GritJavascriptFunctionDefinition;
+use biome_rowan::AstNode;
+#[derive(Debug, Clone, Default)]
+pub(crate) struct FormatGritJavascriptFunctionDefinition;
+impl FormatNodeRule<GritJavascriptFunctionDefinition> for FormatGritJavascriptFunctionDefinition {
+    fn fmt_fields(
+        &self,
+        node: &GritJavascriptFunctionDefinition,
+        f: &mut GritFormatter,
+    ) -> FormatResult<()> {
+        format_verbatim_node(node.syntax()).fmt(f)
```

Comment:
We should implement the formatting while we're at it

Reply:
Done